man/systemd.link.xml

```diff
@@ -691,13 +691,25 @@
       <varlistentry>
         <term><varname>RxBufferSize=</varname></term>
         <listitem>
-          <para>Takes a integer. Specifies the NIC receive ring buffer size. When unset, the kernel's default will be used.</para>
+          <para>Takes an integer. Specifies the NIC receive ring buffer size. When unset, the kernel's default will be used.</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>RxMiniBufferSize=</varname></term>
+        <listitem>
+          <para>Takes an integer. Specifies the NIC receive mini ring buffer size. When unset, the kernel's default will be used.</para>
```

Comment:
hmm, what unit is this in? bytes?

(I understand this is just copied off the explanation of RxBufferSize=, but we should really clarify this here when copying it here.)

If this is bytes, then we really should parse it with parse_size() btw, so thta people can use the usually k, m, g,… suffixes...

Reply:
According to the [kernel](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/ethtool.h?h=v5.6#n488) the unit is "pending entries".

Plus, a lot of, if not most, NICs – I played with bnx2, tg3, and vmxnet3 – round to the nearest supported value, and those are usually relatively sparse, so allowing suffixes would be additionally unhelpful.

Reply:
what's an "entry"? a network packet? if so, should probably say so

Reply:
Updated manpage to specify that the units are pending packets in all four cases